packages/dx-react-grid/src/components/table-layout/virtual-table-layout.jsx

```diff
@@ -34,6 +35,8 @@ export class VirtualTableLayout extends React.PureComponent {
     this.getRowHeight = this.getRowHeight.bind(this);
     this.updateViewport = this.updateViewport.bind(this);
     this.handleContainerSizeChange = this.handleContainerSizeChange.bind(this);
+
+    this.isEdgeBrowser = isEdgeBrowser();
```

Comment:
On the server `this.isEdgeBrowser = false` because if customer use server-side rendering window is `undefined`

And when user will open a page by Edge browser `this.isEdgeBrowser = false`.

NOTE: I can mistake 😢